### PR TITLE
add loading component

### PR DESCRIPTION
**Pull request by christiemolloy**
Adds the loading component to Apicurio.

- Added the HTML manually and had to update some of the styles

Update: got the text to load every time, screenshot below:

Screen Shot 2020-06-02 at 4 13 13 PM

### CODE REVIEW

**Comment by EricWittmann** [comment]
Looks pretty reasonable to me.

**Comment by EricWittmann** [comment]
@christiemolloy sorry I missed the bit about missing text when I first looked at this PR.  I'll have a look when I get a chance and let you know what I find.  We're in the middle of a release of Apicurio Registry right now, so ping me here again if I forget by mid-next week.  :)

File: ui/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@angular/platform-browser-dynamic": "7.2.6",
     "@angular/router": "7.2.6",
     "@patternfly/patternfly": "2.31.6",
+    "@rh-uxd/integration-core": "1.0.2",
```
Comment (dlabaj) [inline ui/package.json:22]: upgrade to 1.0.3

**Comment by dlabaj** [comment]
@EricWittmann Should we add this change to studio as well

**Comment by EricWittmann** [comment]
I'm trying to test this locally but the build is failing for me.  Unrelated to this PR.

**Comment by christiemolloy** [comment]
@EricWittmann oh I updated the package you might need to do an install

**Comment by EricWittmann** [comment]
The problem isn't the new package - and in fact the error is with node-sass, which is an optional dependency.  So despite the VERY ugly error message, it's apparently ignorable.  Classic javascript ecosystem BS.

My bad for not realizing it wasn't actually a fatal error.  :(

**Comment by christiemolloy** [comment]
@EricWittmann ahh yep thats because I added the sass file into the loading screen to get the spinner, so we can ignore the error?

**Comment by EricWittmann** [comment]
Now that I have it running, I can say that the text doesn't always appear for me...

image

I would say it usually does *not* appear for me.

That said - I like it just fine with and without the text. :)

**Comment by EricWittmann** [comment]
And I have no idea at all why not.  Maybe @dlabaj has an idea.

**Comment by EricWittmann** [comment]
I guess we can ignore the error...

**Comment by christiemolloy** [comment]
@EricWittmann and this is with the latest changes I made in this [commit](https://github.com/Apicurio/apicurito/pull/105/commits/12373227c0217df83700a71572cb71f36946f0b9) ?

**Comment by EricWittmann** [comment]
Yes but I'll try a fresh clone and try again.

**Comment by EricWittmann** [comment]
Tried this again on a different machine and now it always works.  So +1 from me.  Will merge.

**Pull request merged**